lib/skate_web/controllers/version_controller.ex

```diff
@@ -0,0 +1,7 @@
+defmodule SkateWeb.VersionController do
+  use SkateWeb, :controller
+
+  def version(conn, _params) do
+    send_resp(conn, 200, Application.get_env(:skate, :version))
```

Comment:
suggestion(non-blocking): IMO it's more useful to get this at compile time than runtime. I dunno about testing here though...
```suggestion
    send_resp(conn, 200, Application.compile_env(:skate, :version))
```

Reply:
I got 👇 error 😭 

```
(RuntimeError) Application.compile_env/3 cannot be called inside functions, only in the module body
```

Reply:
https://github.com/mbta/skate/pull/2549/files/cf37dce0f0efb749a5224ce7f4f892e7717dd15b#r1575385654